### PR TITLE
UIFC 228 replace bigtest by rtl jest

### PR DESCRIPTION
**Pull request by elsenhans**
https://issues.folio.org/browse/UIFC-228

### CODE REVIEW

File: src/components/MetadataCollections/MetadataCollections.test.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        />
+      </StripesContext.Provider>
+    </Router>,
+    translationsProperties
+  )
+);
+
+describe('Collections SASQ View', () => {
+  beforeEach(() => {
+    renderMetadataCollections(metadatacollections, testCollection, tinySources, noop, noop, noop, '', '', noop);
```
Comment (rchr) [inline src/components/MetadataCollections/MetadataCollections.test.js:55]: You can remove the arguments given to `renderMetadataCollections()`.

File: src/components/MetadataSources/MetadataSources.test.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        />
+      </StripesContext.Provider>
+    </Router>,
+    translationsProperties
+  )
+);
+
+describe('Sources SASQ View', () => {
+  beforeEach(() => {
+    renderMetadataSources(metadatasources, testSource, noop, noop, noop, '', '', noop);
```
Comment (rchr) [inline src/components/MetadataSources/MetadataSources.test.js:63]: You can remove the arguments given to `renderMetadataSources()`.

**Review by rchr — CHANGES_REQUESTED** [review]
Looks good! Thank you!
Just two minor issues.

File: src/components/MetadataCollections/MetadataCollections.test.js (hunk 1 above)
Comment (elsenhans) [reply src/components/MetadataCollections/MetadataCollections.test.js]: Thanks! Arguments have been removed.

File: src/components/MetadataSources/MetadataSources.test.js (hunk 2 above)
Comment (elsenhans) [reply src/components/MetadataSources/MetadataSources.test.js]: Thanks! Arguments have been removed.

**Review by rchr — APPROVED** [review]

**Pull request merged**